fMRIVolume/scripts/OneStepResampling.sh

```diff
@@ -92,10 +94,47 @@ ScoutInput=`getopt1 "--scoutin" $@`  # "${15}"
 ScoutInputgdc=`getopt1 "--scoutgdcin" $@`  # "${15}"
 ScoutOutput=`getopt1 "--oscout" $@`  # "${16}"
 JacobianOut=`getopt1 "--ojacobian" $@`  # "${18}"
+fMRIReferencePath=`getopt1 "--fmrirefpath" $@` # "${19}"
+fMRIReferenceReg=`getopt1 "--fmrirefreg" $@`  # "${20}"
+
+# defaults
+fMRIReferencePath=${fMRIReferencePath:-NONE}
+fMRIReferenceReg=${fMRIReferenceReg:-NONE}
+
+# --- Report arguments
+
+verbose_echo "  "
+verbose_red_echo " ===> Running OneStepResampling"
+verbose_echo " "
+verbose_echo " Using parameters ..."
+verbose_echo "         --workingdir: ${WD}"
+verbose_echo "             --infmri: ${InputfMRI}"
+verbose_echo "                 --t1: ${T1wImage}"
+verbose_echo "         --fmriresout: ${FinalfMRIResolution}"
+verbose_echo "         --fmrifolder: ${fMRIFolder}"
+verbose_echo "      --fmri2structin: ${fMRIToStructuralInput}"
+verbose_echo "         --struct2std: ${StructuralToStandard}"
+verbose_echo "              --owarp: ${OutputTransform}"
+verbose_echo "             --oiwarp: ${OutputInvTransform}"
+verbose_echo "       --motionmatdir: ${MotionMatrixFolder}"
+verbose_echo "    --motionmatprefix: ${MotionMatrixPrefix}"
+verbose_echo "              --ofmri: ${OutputfMRI}"
+verbose_echo "--freesurferbrainmask: ${FreeSurferBrainMask}"
+verbose_echo "          --biasfield: ${BiasField}"
+verbose_echo "            --gdfield: ${GradientDistortionField}"
+verbose_echo "            --scoutin: ${ScoutInput}"
+verbose_echo "         --scoutgdcin: ${ScoutInputgdc}"
+verbose_echo "             --oscout: ${ScoutOutput}"
+verbose_echo "          --ojacobian: ${JacobianOut}"
+verbose_echo "        --fmrirefpath: ${fMRIReferencePath}"
+verbose_echo "         --fmrirefreg: ${fMRIReferenceReg}"
+verbose_echo " "
 
 BiasFieldFile=`basename "$BiasField"`
 T1wImageFile=`basename $T1wImage`
 FreeSurferBrainMaskFile=`basename "$FreeSurferBrainMask"`
+ScoutName=`basename "$ScoutInputgdc"`
```

Comment:
Why was this variable added?  i.e., where does it ever get used?

Reply:
It was used in previous iteration. Not used and removed now.